reV/generation/generation.py

```diff
@@ -100,6 +106,102 @@ class Gen:
                  'lcoe_fcr': {'scale_factor': 1, 'units': 'dol/MWh',
                               'dtype': 'float32', 'chunks': None,
                               'type': 'scalar'},
+                 # Solar water heater
+                 'T_amb': {'scale_factor': 1, 'units': 'C',
+                           'dtype': 'int16', 'chunks': None,
+                           'type': 'array'},
+                 'T_cold': {'scale_factor': 1, 'units': 'C',
+                            'dtype': 'float32', 'chunks': None,
+                            'type': 'array'},
```

Comment:
One important thing that I didnt catch before. We always want to output profile data in UTC to match the input resource data. So we roll everything to local time for SAM to do the solar position stuff, then roll it back to UTC for the output data. You can see us doing that in `gen_profile()` below. And you can also see how we intercept data in `collect_outputs()` to make sure this happens. Can you think of a way to intercept all arrays coming out of your new solar thermal classes and roll to UTC? Or actually for all generation classes? 

https://github.com/NREL/reV/blob/6964f5f2d660e6877a51158ee75557549aec7d09/reV/SAM/generation.py#L180-L195
https://github.com/NREL/reV/blob/6964f5f2d660e6877a51158ee75557549aec7d09/reV/SAM/generation.py#L197-L215